src/openrct2/object/RideObject.h

```diff
@@ -50,8 +50,8 @@ class RideObject final : public Object
     void ReadLegacyVehicle(IReadObjectContext* context, OpenRCT2::IStream* stream, rct_ride_entry_vehicle* vehicle);
 
     void ReadJsonVehicleInfo(IReadObjectContext* context, json_t& properties);
-    std::vector<rct_ride_entry_vehicle> ReadJsonCars(json_t& jCars);
-    rct_ride_entry_vehicle ReadJsonCar(json_t& jCar);
+    std::vector<rct_ride_entry_vehicle> ReadJsonCars([[maybe_unused]] IReadObjectContext* context, json_t& jCars);
+    rct_ride_entry_vehicle ReadJsonCar([[maybe_unused]] IReadObjectContext* context, json_t& jCar);
```

Comment:
i assume this is for something in the future?

Reply:
Yes, the feature used to be in this PR but I changed how it was in a later PR so I just removed it from this one.